roles/teku/tasks/setup.yaml

```diff
@@ -51,7 +51,7 @@
           (teku_mev_boost_enabled | ternary(teku_mev_boost_beacon_command,[])),
         [])) +
         (teku_checkpoint_sync_enabled | ternary(
-          teku_container_command_extra_args | map('regex_replace',"--initial-state=.*",teku_container_command_checkpoint_args),
+          teku_container_command_extra_args + teku_container_command_checkpoint_args,
```

Comment:
This will not work for custom networks and my solution will not work for non custom networks. We need a solution that will work for both custom and non custom networks.

Reply:
added a validation that will make it fail if configured wrong. The "ideal" solution is open as an issue on the teku repo.